app/main.js

```diff
@@ -320,13 +319,14 @@ function getTrayMenu () {
     type: 'separator'
   })
 
-  if (!isPaused) {
+  if (breakPlanner == null || !breakPlanner.isPaused) {
```

Comment:
I'm not sure if the null check is necessary here. I checked and it seems as long as the line with `app.on('ready', createTrayIcon)` is after `app.on('ready',loadSettings)` then breakPlanner should already exist.